src/main/java/life/qbic/model/download/QbicDataDownloader.java

```diff
@@ -170,22 +171,24 @@ public void downloadRequestedFilesOfDatasets(
 
         LOG.info(String.format("Number of files found: %s", foundRegexFilteredIDs.size()));
 
-        downloadFilesFilteredByIDs(ident, foundRegexFilteredIDs);
+        //downloadFilesFilteredByIDs(ident, foundRegexFilteredIDs);
```

Comment:
Can we remove that?

Reply:
No, we need a todo statement, regex needs to be fixed in a future release.